src/app/containers/ATIAnalytics/beacon/index.js

```diff
@@ -2,26 +2,31 @@ import sendBeacon from '../../../lib/analyticsUtils/sendBeacon';
 import { buildATIEventTrackUrl } from '../atiUrl';
 
 export const sendEventBeacon = async ({
+  campaignName,
   componentName,
+  format,
+  pageIdentifier,
+  platform,
+  service,
+  statsDestination,
   type,
-  componentInfo,
-  ...params
+  url,
```

Comment:
Big fan of this change and the refactor in general!

Reply:
I think it's much better to show exactly what is being passed down here